lib/ecto/changeset.ex

```diff
@@ -236,6 +236,26 @@ defmodule Ecto.Changeset do
         |> Ecto.Changeset.validate_required(...)
         |> Ecto.Changeset.validate_length(...)
 
+  Besides the basic types which are mentioned above, such as `:boolean` and `:string`, 
+  parameterized types can also be used in schemaless changesets. They implement
+  the `Ecto.ParameterizedType` behaviour and therefore provide an `init/2` function.
+
+  For example, to use `Ecto.Enum` in a schemaless changeset:
```

Comment:
Not sure, but perhaps we should just show how to init the Ecto.Enum type, instead of including the whole flow?

Reply:
I think that's a good idea.

There are two types can be used in schema changesets or schemaless changesets:
+ basic types - `Ecto.Type`
+ parameterized types - `Ecto.ParameterizedType`

We didn't mention `Ecto.Type` when describing the basic types, then maybe it's not necessary to mention `Ecto.ParameterizedType` when describing the parameterized types.

---
But, from another view, we are calling `Ecto.ParameterizedType.init/2`. If we don't mention the relationship between `Ecto.Enum` and `Ecto.ParameterizedType`, then it might be a bit abrupt, which makes the reader wonder "why do I need to call `Ecto.ParameterizedType.init/2`?"